src/System.ComponentModel.TypeConverter/src/System/ComponentModel/CultureInfoConverter.cs

```diff
@@ -434,7 +434,7 @@ private static class CultureInfoMapper
                     {"Lower Sorbian (Germany)", "dsb-DE"},
                     {"Luxembourgish (Luxembourg)", "lb-LU"},
                     {"Macedonian", "mk"},
```

Comment:
This I think needs to change to "North Macedonia" as "mk" is associated with North Macedonia only. 

North Macedonia (ISO country code MK)
.mk, the Internet country code top-level domain for North Macedonia

Reply:
> This I think needs to change to "North Macedonia" as "mk" is associated with North Macedonia only.

Who said that? mk is neutral and can be associated with any country uses the Macedonian language. That is what we did in the desktop.

Reply:
Sorry, I read that as Macedonia instead of Macedonian.